frontend/config/setupTests.js

```diff
@@ -17,3 +19,7 @@ configure({
 });
 
 jest.mock('@openshift/dynamic-plugin-sdk-utils', () => ({}));
+
+
+
+global.TextEncoder = TextEncoder;
```

Comment:
Can we make sure we have newlines at the end of files? I bet our linter isn't catching this because we are not looking at JS files... might be worth looking into.